go/perf/codec-perf-rig/main.go

```diff
@@ -168,9 +168,8 @@ func buildListIncrementally(count uint64, createFn createValueFn) types.Collecti
 }
 
 func readList(c types.Collection) {
-	var outValue types.Value
 	c.(types.List).IterAll(func(v types.Value, idx uint64) {
-		outValue = v
+		_ = v
```

Comment:
The results in the tables above or below that I just included were with the function bodies completely empty. I will submit that as a patch here so then we don't even have to comment on why they are not empty.